docs/structure.rst

```diff
@@ -1,43 +1,139 @@
-
 Structure of a Reactive Charm
------------------------------
+=============================
+
+A reactive charm should be built using layers, with the "top" layer being called
```

Comment:
Typo?  replace "should" with "must" or "should be" with "is".  It's not optional (really) as layer-basic is part of charms.reactive (in essence) ... I guess, unless the charm author wants to completely replace the layer-basic behaviour?